app/server/src/features/organizations/resolvers.ts

```diff
@@ -10,6 +12,38 @@ const toUserId = toGlobalId('User');
 const toEventId = toGlobalId('Event');
 
 export const resolvers: Resolvers = {
+    Query: {
+        async isOrganizer(parent, args, ctx, info) {
+            if (!ctx.viewer.id) {
+                return false;
+            }
+            if (process.env.NODE_ENV === 'development') return true;
+            const email = await Organization.findEmailByViewerId(ctx.viewer.id, ctx.prisma);
+            const bucketName = 'organizers';
+            const fileName = 'list-of-organizers.csv';
+            const storage = new Storage({
+                projectId: process.env.GCP_PROJECT_ID,
+                keyFilename: process.env.GOOGLE_APPLICATION_CREDENTIALS,
+            });
+            const options = {
+                destination: './downloadedFile.txt',
```

Comment:
I think the name of the file should probably be an environment variable, say `ORGANIZER_FILE_PATH`.